### PR TITLE
feat: webpack template generator

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**
feat

**Did you add tests for your changes?**
yet to

**If relevant, did you update the documentation?**
yet to 

**Summary**
Adds a handler to scaffold a webpack repo from the cli

**Does this PR introduce a breaking change?**
no

**Other information**

### CODE REVIEW

File: packages/generators/init-template/webpack-template/prettier.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'es5',
+  arrowParens: 'always',
+};
```
Comment (snitin315) [inline packages/generators/init-template/webpack-template/prettier.config.js:5]: Let's remove it in favor of prettier defaults.

Comment (alexander-akait) [reply packages/generators/init-template/webpack-template/prettier.config.js:5]: Yes, let's do it

Comment (anshumanv) [reply packages/generators/init-template/webpack-template/prettier.config.js]: done

**Comment by alexander-akait** [comment]
Let's covert this to an issue and close, it is too old :disappointed: , but I still think we need it

**Pull request closed without merging**